src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Decentralized social media networks
+description: An overview of decentralized social networks on Ethereum
+lang: en
+template: use-cases
+emoji: ":mega:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Blockchain-based platforms for social interaction and content creation and distribution.  
+summaryPoint2: Decentralized social media networks protect user privacy and enhance data security. 
+summaryPoint3: Ethereum's ERC-20 tokens and NFTs are creating new avenues for content monetization. 
+---
+Today, social media platforms play a huge role in how we communicate and interact with each other. Social networking services have gone beyond mere fads and now underpin the digital economy. 
+
+However, centralized control of these platforms has created many problems. Data breaches, server outages, deplatforming, censorship, privacy violations—these are some of the trade-offs social media users are forced to accept. 
+
+To combat these issues, developers are building novel social media applications on decentralized networks like Ethereum. Decentralized social networks are poised to fix the the ills of legacy social platforms and improve the overall user experience. 
+
+## What are decentralized social networks? {#what-are-decentralized-social-networks}
+Decentralized social media networks are blockchain-based platforms that allow users to exchange information as well as publish and distribute content to audiences. Because these applications run on the blockchain, they are decentralized and resistant to censorship and undue control. 
+
+Many decentralized social networks exist as alternatives to established social media services, such as Facebook, LinkedIn, Twitter, and Medium. But blockchain-powered social networks have a number of features that put them ahead of traditional social platforms. 
+
+## Why do decentralized social media platforms matter? {#decentralized-social-media-benefits}
+
+To understand the benefits of decentralized social media networks, it's important to compare them against traditional social networks:
+
+| Decentralized social media                                                                                                      | Traditional social media                                                                 |
+| ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
+| Information is stored on the blockchain.                                                                                | Information is stored in servers and data centers.                                              |
+| Global distribution of nodes reduces the possibility of outages.                                                        | Highly susceptible to network failure and service downtimes.                                    |
```

Comment:
Sure, they'll definitely have a global distribution of servers. But I think the main difference here is that a blockchain like Ethereum likely has even more servers (i.e., nodes) sustaining the network. Plus, people are _incentivized_ to run these mini-servers--so you can at least bet that some nodes will always be online.

Reply:
> But I think the main difference here is that a blockchain like Ethereum likely has even more servers (i.e., nodes) sustaining the network.

But we're not going to be storing the majority of data on Ethereum. [Livepeer is a good example of this](https://livepeer.org/). I'd bet Livepeer nodes are definitely fewer in number than YouTube servers. Not strongly opposed to keeping it, but I do think the point itself ignores the reality of the current situation.

Reply:
Alright, that's a fair point. Would it be okay to replace the comparison with a line like "As dapps running on Ethereum network, which is sustained by a global, peer-to-peer network of nodes, decentralized social networks are less susceptible to downtime and outages."?

I'm taking a cue from the benefits of dapps listed on the Ethereum.org site: 

_"Once the dapp is live on Ethereum, it will only go down if Ethereum itself goes down. Networks of Ethereum's size are notoriously difficult to attack."_